packages/java-parser/src/productions/expressions.js

```diff
@@ -462,25 +463,30 @@ function defineRules($, t) {
     ]);
   });
 
+  // https://docs.oracle.com/javase/specs/jls/se11/html/jls-15.html#jls-15.10.1
```

Comment:
I personally find these links to the spec a little confusing
because this does not align to the spec style strongly.

But I probably won't be maintaining this code in the long term so as this is subjective
You should decide if to keep the links, remove them or add a disclaimer to them. 😄